Changelog.md

```diff
@@ -2,6 +2,9 @@
 
 ### Ignition Gazebo 4.X.X
 
+1. Add window focus upon mouse entering the render window
+    * [Github pull request 97](https://github.com/ignitionrobotics/ign-gazebo/pull/97)
```

Comment:
dome has not been release yet :) can you move this entry down to the 4.0.0 section?